uat/testing-features/src/main/java/com/aws/greengrass/steps/NetworkSteps.java

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package com.aws.greengrass.steps;
+
+import com.aws.greengrass.NetworkUtils;
+import io.cucumber.java.Before;
+import io.cucumber.java.en.Given;
+import io.cucumber.java.en.When;
+
+import java.io.IOException;
+
+public class NetworkSteps {
+
+    @Before
+    public void connectToNetwork() throws IOException, InterruptedException {
```

Comment:
If a test fails while we change the IPTables and we do not reconnect all other tests will fail